src/components/CourseForm/components/CreateAuthor/styles.module.css

```diff
@@ -0,0 +1,6 @@
+.newAuthorContainer {
+	display: flex;
+	align-items: flex-end;
+	gap: 20px;
+	margin-bottom: 64px;
```

Comment:
The same regarding spacing. Please check it.

Reply:
all ok